### PR TITLE
add week 4 updates for warpsql

### PR DESCRIPTION
**Pull request by PiyushRaj927**
## Project

WarpSQL

### Changes

Week 4 updates

### CODE REVIEW

**Comment by PiyushRaj927** [comment]
Hi @Apoorvgarg-creator can you review this when you have a chance? Thanks!

File: docs/2023/WarpSQL/Plugins, Disaster Recovery, Benchmarking/updates/2023-07-28.md
Comment (Apoorvgarg-creator) [inline docs/2023/WarpSQL/Plugins, Disaster Recovery, Benchmarking/updates/2023-07-28.md:1]: @PiyushRaj927 Please add presentation link

**Review by Apoorvgarg-creator — CHANGES_REQUESTED** [review]

File: docs/2023/WarpSQL/Plugins, Disaster Recovery, Benchmarking/updates/2023-07-28.md
Comment (PiyushRaj927) [reply docs/2023/WarpSQL/Plugins, Disaster Recovery, Benchmarking/updates/2023-07-28.md:1]: Done @Apoorvgarg-creator

File: docs/2023/WarpSQL/Plugins, Disaster Recovery, Benchmarking/updates/2023-07-28.md [hunk 1]
```diff
@@ -11,4 +11,8 @@ author: Piyush Raj
 - [AWS warpSQL deployment #98](https://github.com/Samagra-Development/WarpSQL/pull/98)
 
 ## Learnings
-I learned about AWS Identity and Access Management (IAM) and OIDC for security and access control. I also learned how to use the AWS CLI to delete snapshots and deregister AMI.
\ No newline at end of file
+I learned about AWS Identity and Access Management (IAM) and OIDC for security and access control. I also learned how to use the AWS CLI to delete snapshots and deregister AMI.
+
+slide for the midpoint demo showcase: https://docs.google.com/presentation/d/13-oQPURoz-XgenC-2GlqVa7wnSYXmin8/
```
[the remark this answers is not in the document]
Comment (PiyushRaj927) [reply docs/2023/WarpSQL/Plugins, Disaster Recovery, Benchmarking/updates/2023-07-28.md]: Done @jayesh212

**Review by jayesh212 — APPROVED** [review]

**Pull request merged**